smaily-for-wordpress.php

```diff
@@ -151,16 +151,26 @@ function smaily_subscribe_callback() {
 
 	$array = array_merge( $array, $form_values );
 	require_once( BP . DS . 'code' . DS . 'Request.php' );
-	$request = new Wp_Sendsmaily_Request( $server, $array );
+	$request = new Wp_Smaily_Request( $server, $array );
 	$result  = $request->exec();
 
 	if ( empty( $result ) ) {
-		echo esc_html__( 'Something went wrong', 'wp_sendsmaily' );
+		echo esc_html__( 'Something went wrong', 'wp_smaily' );
```

Comment:
Replace `esc_html__( ... )` with `__( ... )`. Otherwise value will be encoded twice, and it is OK to remove encoding from here because Javascript uses `jQuery.text` function to set `p.error` element content which encodes HTML-entities.